less/index.less

```diff
@@ -49,11 +49,226 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+@ebony-color: #26303C;
+@light-grey: lightgrey;
+
+.center() {
+	display:flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.reset-text() {
+	font-size: 1.6rem;
+}
+
+.font-styles() {
```

Comment:
I like the idea to use this as a function as well. That way, if you ever want to change font, you only have to change it in this one place.